contents/teams/surveys/objectives.mdx

```diff
@@ -1,79 +1,39 @@
+### 🤖 Goal 2: Better Integrate Max with Survey Responses
 
-> Why: Increasing awareness and adoption of Surveys through focused outreach.
-
-> Who: <TeamMember name="Manoel Aranda Neto" photo /> and the [Brand & Vibes Team](/teams/brand).
-
-We're launching two major updates this quarter, new pricing and deeper AI integration. Our core focus is to drive usage, validate product–market fit, and grow our customer base and revenue. However, many users still aren’t aware that surveys are part of our offering. That’s why we’re doubling down on marketing, to reach existing users who haven’t tried Surveys yet, and to attract entirely new customers who could benefit from them.
-
--   📢 Announcements (Collab with Joe)
-    -   New pricing updates
-    -   Flutter surveys
-    -   Fullscreen surveys
-    -   Email-based surveys
-    -   AI-powered contextual survey creation and launching
--   📝 Changelog & Early Access (Surveys Team)
-    -   Ensure all new features are listed in the changelog
-    -   Enroll eligible features under early access and notify interested users
--   📚 Documentation & Tutorials (Collab with Ian)
-    -   Propose 3 new tutorial ideas to the docs team to improve survey onboarding and usage
--   📈 Growth & Adoption (Collab with Brian)
-    -   Explore the use of paid ads or promotional campaigns to drive awareness and adoption of the Surveys product
-
-# 🛠️ Goal 3: Customization
-
-> Why: Expanding the flexibility of Surveys to meet diverse user needs.
+> Why: Max can analyze survey responses when you're viewing them, but can't access them from other conversations.
 
 > Who: <TeamMember name="Lucas Faria" photo />
 
-Surveys already offer a high degree of customization, but our customers continue to ask for even more flexibility. Last quarter, we introduced several enhancements in this area, and we’re now taking it further by addressing the remaining gaps. This update focuses on expanding customization options to better meet the diverse needs of our users and give them even more control over how surveys look and behave. With these improvements, users will have more ways to tailor the survey experience to fit their brand, product flow, and audience.
-
--   [Better wait period controls](https://github.com/PostHog/posthog/issues/30379)
--   Alignment customization [here](https://github.com/PostHog/posthog/issues/28479) and [here](https://github.com/PostHog/posthog/issues/28477)
--   [Inline surveys](https://github.com/PostHog/posthog/issues/30097)
--   [Add icons](https://github.com/PostHog/posthog/issues/28478)
--   [Templated strings](https://github.com/PostHog/posthog/issues/26620)
--   ...
--   This is not an exhaustive list
-
-# 🧩 Goal 4: Deeper integration with Surveys and AI
+Enable Max to access and reference survey responses from any conversation, not just when viewing a specific survey. Users should be able to ask "What are users saying about feature X?" or "Summarize my NPS feedback" from anywhere in PostHog.
```

Comment:
How are survey responses taken into account for the Array product? Is that separate from Max (RIP) or the same thing?

```suggestion
Enable PostHog AI to access and reference survey responses from any conversation, not just when viewing a specific survey. Users should be able to ask "What are users saying about feature X?" or "Summarize my NPS feedback" from anywhere in PostHog.
```

Reply:
it's a separate thing. Goal 2 is about making PostHog AI able to leverage survey responses on any question that is asked, if it's relevant.

The recording/Array product is a separate thing, disjoint for surveys atm. Our goal will be instead to get signals/tasks from team meetings/user interviews.